lib/MusicBrainz/Server/Form/Role/EditNote.pm

```diff
@@ -18,9 +19,13 @@ has_field 'make_votable' => (
 after validate => sub {
     my $self = shift;
 
-    if ($self->requires_edit_note && (!defined $self->field('edit_note')->value || $self->field('edit_note')->value eq '')) {
```

Comment:
The `''` check was removed, but it doesn't look like `is_valid_edit_note` checks for an empty string either. (Or is it just always undefined when empty? Unclear why the check was removed anyway.)

Reply:
AFAICT it's just undefined if empty. I tried to enter a string, then remove it, then submit again, and I still get the "You must provide an edit note" message. (I'd love to add a period to that string but breaking the translation for it feels silly).

Reply:
I see, seems to be documented at https://metacpan.org/pod/HTML::FormHandler::Field#not_nullable